### DISCUSSION TITLE
Payment Providers

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#172] [Q&A]
What payment providers do people think need to be supported. The current SS site support Paypal only.

I know Zelle is big in some places, as is Square. Others for use in EU/APAC?

**Comment by SQLClause** [comment]
Can Google Pay or Apple Pay work as well you think?

**Comment by suxstellino** [comment]
There are many gateways that supports many payment methods, like stripe https://stripe.com/about. As far as I know, the API is super simple. A friend of mine have also a sample private repo (hopefully I'll link it in a reply)

**Comment by spaghettidba** [comment]
Payment means the event is not free anymore, so not allowed to use sessionize or other tools for free. Certainly not a problem for many events.

**Comment by steverezhener** [comment]
@spaghettidba , Steve is talking about payments from event sponsors. while some events are charging to cover lunch cost, no events are charging fees for access, so sessionize is going to be happy.

[a comment by an automated account is not shown]

**Comment by steverezhener** [comment]
@way0utwest , while Zelle and Venmo would be nice (they usually have no fees), most if not all sponsors are paying with a corporate credit card (or personal credit card with company reimbursement). In that case, PayPal is better. It's cheaper than Google Pay, Apple Pay, and Stripe for non-profits.
https://www.paypal.com/us/smarthelp/article/how-much-does-paypal-cost-for-nonprofits-and-charities-faq4103
Qualified 501(c)(3) organizations can receive a discounted charity rate of 2.2% + $0.30 per domestic transaction.* Get started at paypal.com/charities.

[a comment by an automated account is not shown]
